data_structures/linked-list/node.py

```diff
@@ -0,0 +1,19 @@
+
+class Node:
+    """Class for a node."""
+    def __init__(self, val, next=None):
+        """Initialize the node instance."""
+        self.val = val
+        self._next = next
+
+    def __str__(self):
+        """Return the value as a string."""
+        return '{val}'.format(val=self.val)
```

Comment:
Since the only thing in this string is the value of the node, you can just return `str(self.val)`.